app/views/dashboard/assessmentDetails.scala.html

```diff
@@ -33,10 +32,36 @@
     </tr>
 }
 
-@table(list: Seq[(String, ApiAssessment)], tableId: String, isDraft: Boolean = false) = {
-    <div id="viewAssessmentRadioGroup_@tableId" class="form-group">
+@effectiveDate(assessment: ApiAssessment) = @{
+    assessment.effectiveDate.map(Formatters.formatDate).getOrElse("")
+}
+
+@fromDate(assessment: ApiAssessment) = @{
+    assessment.currentFromDate.map(Formatters.formatDate).getOrElse("")
+}
+
+@endDate(assessment: ApiAssessment, isDraft: Boolean) = @{
+    if(!isDraft) {
+        (assessment.currentFromDate, assessment.currentToDate) match {
+            case (Some(_), None)     => Some(Messages("assessments.enddate.present"))
+            case (Some(_), Some(to)) => Some(Formatters.formatDate(to))
+            case _                   => None
+        }
+    } else None
+}
+
+@ariaLabel(assessment: ApiAssessment, listType: String, isDraft: Boolean) = @{
+    val listTypeText = listType.takeWhile(_ != 'A').capitalize
```

Comment:
what does this line do?

Reply:
`listType.takeWhile(_ != 'A').capitalize`

Reply:
Splits the input "currentAssessments" (for example) to "Current"